front/lib/resources/trigger_resource.ts

```diff
@@ -112,6 +113,43 @@ export class TriggerResource extends BaseResource<TriggerModel> {
     return this.baseFetch(auth);
   }
 
+  static async listByUserEditor(auth: Authenticator) {
+    const user = auth.getNonNullableUser();
+
+    return this.baseFetch(auth, {
+      where: {
+        editor: user.id,
+      },
+    });
+  }
+
+  static async listByUserSubscriber(auth: Authenticator) {
+    const workspace = auth.getNonNullableWorkspace();
+    const user = auth.getNonNullableUser();
+
+    const res = await this.model.findAll({
```

Comment:
FYI I tried to use baseFetch on this one too, but couldn't get it to work with the complex query.